src/validation.cpp

```diff
@@ -20,6 +20,7 @@
 #include "init.h"
 #include "policy/fees.h"
 #include "policy/policy.h"
+#include "policy/rbf.h"
```

Comment:
Do we want validation to depend on those policy headers? It seems the wrong way around.